### PR TITLE
Integrate .readthedocs.yaml, done using https://github.com/zopefounda…

### PR DESCRIPTION
**Pull request by dhavlik**
…tion/meta/pull/205.

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Sphinx>4.3
+sphinx-rtd-theme>=1
+ZConfig
+Zope
```
Comment (icemac) [inline docs/requirements.txt:4]: This looks strange.

**Review by icemac — CHANGES_REQUESTED** [review]

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Sphinx
+repoze.sphinx.autointerface
```
Comment (icemac) [inline docs/requirements.txt:2]: There is a doc-requirements.txt in the root of the repository.

**Review by icemac — CHANGES_REQUESTED** [review]

**Review by icemac — APPROVED** [review]

File: docs/conf.py [hunk 3]
```diff
@@ -118,7 +118,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx_rtd_theme'
```
Comment (dataflake) [inline docs/conf.py:121]: If this theme change from `sphinx_rtd_theme` to `default` was in response to build errors: The version of `sphinx_rtd_theme` installed by RTD is ancient and you must force a higher version in `docs.requirements.txt`. Also, as documented in the Zope versions file, `sphinx_rtd_theme` requires `docutils<0.19`. The following entries into `docs/requirements.txt` fixed all these issues for me:

```
docutils<0.19
sphinx_rtd_theme>1
```

Comment (icemac) [reply docs/conf.py:121]: Done in https://github.com/zopefoundation/BTrees/commit/46b78172219e84d9dfa2c104d05586958b4f3056.

**Pull request merged**